backend/game/cards/master_card_list.py

```diff
@@ -1,28 +1,26 @@
 # Energy Manipulation Cards
 from game.cards.discard_cards.energy_manipulation_cards.energize import Energize
-from game.cards.keep_cards.energy_manipulation_cards.energy_hoarder import EnergyHoarder
-from game.cards.keep_cards.energy_manipulation_cards.friend_of_children import FriendOfChildren
-from game.cards.keep_cards.energy_manipulation_cards.solar_powered import SolarPowered
-from game.cards.keep_cards.energy_manipulation_cards.were_only_making_it_stronger import WereOnlyMakingItStronger
-
 # Health Manipulation cards
 from game.cards.discard_cards.health_manipulation_cards.fire_blast import FireBlast
 from game.cards.discard_cards.health_manipulation_cards.high_altitude_bombing import HighAltitudeBombing
-
 # Multi-Manipulation Cards
 from game.cards.discard_cards.multi_manipulation_cards.gas_refinery import GasRefinery
 from game.cards.discard_cards.multi_manipulation_cards.jet_fighters import JetFighters
 from game.cards.discard_cards.multi_manipulation_cards.national_guard import NationalGuard
 from game.cards.discard_cards.multi_manipulation_cards.nuclear_power_plant import NuclearPowerPlant
 from game.cards.discard_cards.multi_manipulation_cards.tanks import Tanks
 from game.cards.discard_cards.multi_manipulation_cards.vast_storm import VastStorm
-
 # Victory Point Manipulation Cards
 from game.cards.discard_cards.victory_point_manipulation_cards.apartment_building import ApartmentBuilding
 from game.cards.discard_cards.victory_point_manipulation_cards.commuter_train import CommuterTrain
 from game.cards.discard_cards.victory_point_manipulation_cards.corner_store import CornerStore
 from game.cards.discard_cards.victory_point_manipulation_cards.evacuation_orders import EvacuationOrders
 from game.cards.discard_cards.victory_point_manipulation_cards.skyscraper import Skyscraper
+from game.cards.keep_cards.energy_manipulation_cards.energy_hoarder import EnergyHoarder
+from game.cards.keep_cards.energy_manipulation_cards.friend_of_children import FriendOfChildren
+from game.cards.keep_cards.energy_manipulation_cards.solar_powered import SolarPowered
+from game.cards.keep_cards.energy_manipulation_cards.were_only_making_it_stronger import WereOnlyMakingItStronger
+from game.cards.keep_cards.turn_manipulation_cards.GiantBrain import GiantBrain
```

Comment:
Some weird import movements here that seem to be due to probably resolves

Reply:
It's from save actions organizing imports automatically